config/config.go

```diff
@@ -77,6 +77,7 @@ type SigningProfile struct {
 	NameWhitelistString string     `json:"name_whitelist"`
 	AuthRemote          AuthRemote `json:"auth_remote"`
 	CTLogServers        []string   `json:"ct_log_servers"`
+	AllowedExtensions   []OID      `json:"allowed_extensions"`
```

Comment:
Actually, that's how I did it in my initial PR, and I think that's *not* the way to go here.  Instead of copying extensions from the CSR, we should have the calling code provide its own extensions.  That gives the calling code a lot more flexibility in what it can add, and avoids taking extension data from an untrusted source.